setup.py

```diff
@@ -19,10 +19,6 @@
 
 from mesonbuild.coredata import version
 
-if sys.version_info[0] < 3:
```

Comment:
This should probably stay, in case people have old versions of pip that don't understand `python_requires`.

Reply:
Yes.

Reply:
Updated to check for 3.5.0 instead.